ppa/archive/templates/archive/snippets/search_form.html

```diff
@@ -1,6 +1,19 @@
 {% load semanticui humanize static %}
 <form class="ui form{% if error %} error{% endif %}">
     <div class="ui vertical stackable grid segment">
+        {# include hidden cluster field value for form submission #}
+        {{ search_form.cluster }}
+
+        {% if search_form.cluster.value %}
+        <div class="twelve wide column">
+            {# TODO: icon, text and remove filter here  #}
+            <div class="cluster-work icon container" >
+                You are searching and browsing within a cluster.
```

Comment:
Here's where "cluster" is forward-facing term rather than just internal. Just to flag in case we end up changing the public rhetoric of it.